docs/src/tutorials/standalone/Usage/LSM_single_column_tutorial.jl

```diff
@@ -1,5 +1,5 @@
 # The `AbstractModel`
-# [tutorial](@ref "General setup")
+# [tutorial](@ref "Hydrostatic Equilibrium test for Richards Equation")
```

Comment:
I think we want to refer to this one (which also needs a title?) https://clima.github.io/ClimaLand.jl/stable/generated/standalone/Usage/model_tutorial/

Reply:
the title could be "Creating a Model"?